test/framework/integration_framework/integration_test_framework.cpp

```diff
@@ -245,8 +245,9 @@ IntegrationTestFramework::IntegrationTestFramework(
   config_.max_proposal_size = 10;
   config_.mst_support = mst_support;
 
-  db_wsv_path_ = (fs::temp_directory_path() / fs::unique_path()).string();
-  db_store_path_ = (fs::temp_directory_path() / fs::unique_path()).string();
+  db_wsv_path_ = (fs::temp_directory_path() / "iroha_temporary_wsv").string();
```

Comment:
I think unique_path is some random value to run tests in parallel, isn't it. Look like now tests could share access to same file when running concurrently..